client/client.go

```diff
@@ -67,6 +68,28 @@ type TargetConf struct {
 	macros map[string]string
 }
 
+type TerminalConf struct {
+	Color bool
+}
+
+//Used for color print in terminal base on color flag
+//in configuration file
+func (clr TerminalConf) Printf(c string, s string, a ...interface{}) {
+	if clr.Color {
+		fmt.Printf("\x1b["+c+"m"+s+"\x1b[0m", a...)
```

Comment:
first of all if we have various highlighting in same line then we will call function for separately for all color and the and there is color attribute in function which consist of two things color+ text decoration . so it is not a problem . 
and yes if color flag in configuration is off then it will print the simple text.